<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>droneFormWindow</class>
 <widget class="QMainWindow" name="droneFormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_ID">
          <property name="text">
           <string>ID :</string>
          </property>
          <property name="buddy">
           <cstring>txt_ID</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txt_ID">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_wingType">
          <property name="text">
           <string>Wing Type :</string>
          </property>
          <property name="buddy">
           <cstring>cBox_wingType</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cBox_wingType">
          <item>
           <property name="text">
            <string>Fixed</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rotor</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_maxAngle">
          <property name="text">
           <string>Max Angle :</string>
          </property>
          <property name="buddy">
           <cstring>txt_maxAngle</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="txt_maxAngle"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lbl_maxAltitude">
          <property name="text">
           <string>Max Altitude :</string>
          </property>
          <property name="buddy">
           <cstring>txt_maxAltitude</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="txt_maxAltitude"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lbl_maxRange">
          <property name="text">
           <string>Max Range :</string>
          </property>
          <property name="buddy">
           <cstring>txt_maxRange</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="txt_maxRange"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lbl_cruiseSpeed">
          <property name="text">
           <string>Cruise Speed :</string>
          </property>
          <property name="buddy">
           <cstring>txt_cruiseSpeed</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="txt_cruiseSpeed"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lbl_Autonomy">
          <property name="text">
           <string>Autonomy :</string>
          </property>
          <property name="buddy">
           <cstring>txt_Autonomy</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="txt_Autonomy"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lbl_Payload">
          <property name="text">
           <string>Payload :</string>
          </property>
          <property name="buddy">
           <cstring>txt_Payload</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="txt_Payload"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="lbl_groundPosition">
          <property name="text">
           <string>Ground Position :</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="lbl_latitude">
          <property name="text">
           <string>- Latitude :</string>
          </property>
          <property name="buddy">
           <cstring>txt_latitude</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="txt_latitude"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="lbl_longitude">
          <property name="text">
           <string>- Longitude :</string>
          </property>
          <property name="buddy">
           <cstring>txt_longitude</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="txt_longitude"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="txt_errors">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_Save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
